ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -523,10 +523,11 @@ private void SetupReactContext(ReactContext reactContext)
             rootView.TouchHandler = new TouchHandler(rootView);
 
             var jsAppModuleName = rootView.JavaScriptModuleName;
+            var initialProps = rootView.InitialProps;
```

Comment:
Will do! I tend to follow by example where possible - going by your advice, the jsAppModuleName variable should also be in-line.